modules/admin_manual/pages/enterprise/authentication/kerberos.adoc

```diff
@@ -393,7 +371,48 @@ samba-tool domain exportkeytab --principal=HTTP/<FQDN> <keytab-file-location>/<u
 ----
 --
 
-* Protect the `keytab` file so only the owner (the webserver) can read it:
+. Move the generated keytab file `<user-name>.keytab` to the Linux server hosting the ownCloud instance to location `<keytab-file-location>`. Note that the file must be accessible by the web server.
+
+== Configure ownCloud Server
+
+Follow these steps on the server running ownCloud:
+
+. Configure `/etc/krb5.conf` for use with Kerberos. +
+Note that only required/recommended or non-default settings are used:
++
+--
+[source,plaintext]
+----
+[libdefaults]
+    default_realm        = <REALM>
+    default_tkt_enctypes = aes256-cts-hmac-sha1-96
+    default_tgs_enctypes = aes256-cts-hmac-sha1-96
+    permitted_enctypes   = aes256-cts-hmac-sha1-96
+    forwardable          = true
+    proxiable            = true
```

Comment:
This can be removed IMHO. @jvillafanez ?

Reply:
I have read tons of documents pointingto the need for enctypes for security reasons, I would like to know in detail why this now shoud not be needed anymore??

Reply:
I have removed the proxiable parameter in the meanwhile. We can readd it at any time if really needed.